code/controllers/subsystem/puppetmaster.dm

```diff
@@ -0,0 +1,117 @@
+var/datum/subsystem/puppetmaster/SSpuppetmaster
+
+/datum/subsystem/puppetmaster
+	name = "Puppetmaster"
+	priority = 0
+
+	var/list/puppets = list()
+	var/antags = 0
+
+/datum/subsystem/puppetmaster/New()
+	NEW_SS_GLOBAL(SSpuppetmaster)
+
+/datum/subsystem/puppetmaster/Initialize()
+	. = ..()
+
+/datum/subsystem/puppetmaster/stat_entry()
+	..("Puppets:[puppets.len]|Antags:[antags]")
```

Comment:
Will later have an objective tally and/or alive count, because at certain thresholds, it'll wake up the Kingmaker subsystem again.